HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/AppCoordinator.swift

```diff
@@ -67,6 +67,7 @@ final class AppCoordinator: Coordinator {
     }
     
     func start() {
+        try! apiLib.removeStoredCredentials()
```

Comment:
Did you commit this by accident? If we keep this in the example, then clients might think they should always call this. This would cause new users to be generated every time.

This method should be called if the client id or base url changed or there is some issue regarding authentication.

Reply:
I wanted to do it -> I'll comment out this line and leave a comment for it

Reply:
I've decided to remove it 🙈